elyra/pipeline/processor_local.py

```diff
@@ -169,7 +170,7 @@ def __init__(self, root_dir: str):
         super(NotebookOperationProcessor, self).__init__(root_dir)
 
     def _create_execute_command(self, filepath: str, cdw: str) -> list:
-        return ['papermill', filepath, filepath, '--cwd', cdw]
+        return ['papermill', filepath, filepath, '--cwd', cdw, '--no-progress-bar']
```

Comment:
Why would we remove the `progress-bar`?

Reply:
It didn't seem to provide much value.  I had removed it prior to discovering that `stderr` should be decoded and it produced this kind of output:
`
RuntimeError: Internal error executing /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb: b'Input Notebook:  /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb\nOutput Notebook: /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb\nWorking directory: /Users/kbates/notebooks/elyra/pipeline1\n\rExecuting:   0%|          | 0/2 [00:00<?, ?cell/s]Executing notebook with kernel: python3\n\rExecuting:  50%|\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88     | 1/2 [00:01<00:01,  1.28s/cell]\rExecuting: 100%|\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88\xe2\x96\x88| 2/2 [00:01<00:00,  1.19cell/s]\n
`

I just ran with it enabled (and `stderr` decoded) and it produces this:
```
RuntimeError: Internal error executing /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb: Input Notebook:  /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb
Output Notebook: /Users/kbates/notebooks/elyra/pipeline1/node1.ipynb
Working directory: /Users/kbates/notebooks/elyra/pipeline1
Executing:   0%|          | 0/2 [00:00<?, ?cell/s]Executing notebook with kernel: python3
Executing:  50%|âââââ     | 1/2 [00:01<00:01,  1.21s/cell]
Executing: 100%|ââââââââââ| 2/2 [00:01<00:00,  1.24cell/s]
```
I don't really see the value in seeing the progress bar either way.  We will always expect 100% completion of each node, so I figured its inclusion wasn't "a feature".   I'm okay restoring the progress bar if that's what we want.  With these changes, this stuff will now reach the client - which I figured was what this issue was driving at.

Reply:
I've restored the progress bar per side discussion.